pkg/component/monitoring/charts/bootstrap/aggregate-prometheus-rules/seed.rules.yaml

```diff
@@ -0,0 +1,16 @@
+groups:
+- name: seed.rules
+  rules:
+  - alert: PodStuckInPending
+    expr: |
+      sum_over_time(kube_pod_status_phase{phase="Pending"}[5m]) > 0
+    for: 10m
+    labels:
+      severity: warning
+      type: seed
+      visibility: operator
+    annotations:
+      summary: Some pod is stuck in pending
+      description: >
+        Pod in namespace {{$labels.namespace}} was stuck
```

Comment:
Should we specify which pod, using `{{$labels.pod}}`? Or did you go with just the namespace for readability?